README.md

```diff
@@ -77,6 +77,11 @@ previous stable release.
 This module targets Foreman 3.1+.
 The module can not be used to manage Foreman installations on EL7.
 
+This module configures Apache to serve static assets from
+`/var/lib/foreman/public` directly, which requires an appropriate
+SELinux policy. Such a policy was introduced in [`foreman-selinux`
+version 3.5](https://projects.theforeman.org/issues/35402).
```

Comment:
Additionally, plugins need some work. To use the old style, set `foreman::config::apache::proxy_no_proxy_uris` to not include `/assets`.

Or something along those lines.